koku/api/provider/provider_manager.py

```diff
@@ -265,6 +265,6 @@ def provider_post_delete_callback(*args, **kwargs):
         delete_func = partial(delete_archived_data.delay, provider.customer.schema_name, provider.type, provider.uuid)
         transaction.on_commit(delete_func)
 
-    refresh_materialized_views.s(
+    refresh_materialized_views(
```

Comment:
Is this only called during the source delete which is now itself async?

Reply:
Yes, that is only called during the source delete.